addons/test_mail/tests/test_mail_followers.py

```diff
@@ -122,6 +122,14 @@ def test_followers_multiple_subscription_update(self):
         test_record.message_subscribe(partner_ids=[self.user_employee.partner_id.id], subtype_ids=[self.mt_cl_def.id, self.mt_al_def.id])
         self.assertEqual(follower.subtype_ids, self.mt_cl_def | self.mt_al_def)
 
+    def test_followers_inverse_message_partner(self):
+        test_record = self.test_record.with_user(self.user_employee)
+        test_partner = self.env['res.partner'].create({'name': 'partner_a',
+                                                       'email': 'test@test.lan'})
+        self.test_record.message_partner_ids = [test_partner.id]
```

Comment:
You can also test some commands to remove m2m entries, force update with several entries (check that when we add old entries are kept, or that when we replace all entries old entries are not kept), ... a test with a single ID is the simplest one, better check more complex cases if possible :)

Reply:
I increased coverage, I was going to add a test to make sure we don't bypass access rights by going through partners instead of followers but does that make sense? I can't really find any similar test so there might be a reason